### PR TITLE
Portifolio files branch

### PR DESCRIPTION
**Pull request by iankiema**
Added and updated the following files:
-Style.css
-README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -242,7 +242,7 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
                 </span>
                 <input  type="email" placeholder="Email address">
                 <textarea name="comments" id="" cols="30" rows="3" placeholder="Write me something..."></textarea>
```
Comment (V-Blaze) [inline index.html]: - [ ] You have done great so far. Your design looks good but there are some project requirements you missed. :+1: .

> Collect data by using the [Formspree service](https://formspree.io/html).
- - According to the project requirement, you are supposed to have a post method on your form with an action that collects the form data and sends to Formspree, :+1: I'll suggest you follow the Formspree link and create an account, then get your lint and add to your from action. :+1: :+1: 

>A user cannot submit the form if one or more of the fields is empty.
- - According to to the requirements, all the input fields are required which means all the inputs need to have the `required` attribute, I'll suggest you add this attribute. :+1: 

> Name field accepts up to 30 chars. Message field accepts up to 500 chars.
- - According to this requirement, the name field and the message input should have a maximum character attribute `maxlength`, I'll suggest you add these to the correct inputs. :+1: :smile:

File: index.html [hunk 2]
```diff
@@ -242,7 +242,7 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
...
-                <button>Get in touch</button>
+                <button class="contact-btn" type="button">Get in touch</button>
```
Comment (V-Blaze) [inline index.html]: - [ ] This is a button to submit the form inputs and therefore following the best practices, it should have a type of `submit`, I'll suggest you update this. :+1:

File: index.html [hunk 3]
```diff
@@ -242,7 +242,7 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
...
             </form>
         </div>
```
Comment (V-Blaze) [inline index.html:236]: - [ ] You did great using a `ul` and li items to lay out the social links on the website footer, however, following the best practices, I'll suggest you fill in the correct alt text on the image tags, it helps screen readers identify each image correctly :+1: :smile: 
Screenshot from 2023-06-13 13-41-11

- -

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @iankiema, @the-faizmohammad  

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

almost-there


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- UI looks great✔
- Clean and organized code👋 ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -235,14 +235,15 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
...
             </div>
 
-            <form class="contact-form" action="">
-                <input class="full-name" type="text" placeholder="Full name">
+            <form class="contact-form" action="https://formspree.io/f/mayzrnlw" method="POST">
+                <input class="full-name" type="text" placeholder="Full name" maxlength="30" required>
                 <span>
-                <input class="first-name" type="text" placeholder="First Name"> <input class="second-name" type="text" placeholder="Last Name">
+                <input class="first-name" type="text" placeholder="First Name" maxlength="30" required>
+                <input class="second-name" type="text" placeholder="Last Name" maxlength="30" required>
```
Comment (joskalenda) [inline index.html]: - [ ] How many input tags are we supposed to have here? I think it is 3 ma, the `name, email, and the message(text area)`, in your case I think you have 2 more additional input tags and currently causing the form not to submit, see?

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @iankiema , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Good design 
- [ ].. ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @iankiema , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Integrated with [formspree service](https://formspree.io/html) ✔️
- Form validated✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**